test/integration/full-protocol/Loans.pairs.fork.t.sol

```diff
@@ -178,16 +184,33 @@ contract WBTCUSDC_ArbiSep_LoansForkTest is WETHUSDC_ArbiSep_LoansForkTest {
     }
 }
 
-//// load existing
-//
+////// load existing sepolia deployment
```

Comment:
This doesn't work currently because this is an older version of the contracts (before the fix review changes).